packages/xo-vmdk-to-vhd/src/vmdk-generate.js

```diff
@@ -122,7 +122,7 @@ ddb.geometry.cylinders = "${geometry.cylinders}"
     assert.strictEqual(buffer.length, grainSizeBytes)
     assert.strictEqual(lbaBytes % grainSizeBytes, 0)
     const markerOverHead = 12
-    const compressed = zlib.deflateSync(buffer, { level: 9 })
+    const compressed = zlib.deflateSync(buffer, { level: 0 })
```

Comment:
This completely disables the compression, using `zlib.constants.Z_BEST_SPEED` (i.e. `1`) might be better.